recipes/pzflow/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "pzflow" %}
+{% set version = "3.0.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/jfcrenshaw/pzflow/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 9c3142f43d23b217301e9f226573593d6418bb9e4fbb40c4565af837675a104e
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3.8
+    - pip
+    - poetry
+  run:
+    # I don't think any of these constraints are real but here I am
+    - python >=3.8
+    - jax =0.4.2
+    - optax =0.1.4
+    - pandas >=1.1
+    - dill =0.3.6
+    - tqdm =4.64.1
+
+test:
```

Comment:
Does pip check not work?

Reply:
let's test it!

Reply:
I ignore pip check out of habit. When it passes, that's fine. When it fails, we ignore it. So why bother?

Reply:
I added it to see if it passes here.

Reply:
I guess, yeah. I find it helpful to see if something fails down the line. Most of the time, it's nothing we can control, but sometimes it's a regression in a feedstock, and then it's a nice trip wire.